modules/vaos/app/services/vaos/middleware/vaos_logging.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+module VAOS
+  module Middleware
+    class VaosLogging < Faraday::Middleware
+      def initialize(app)
+        super(app)
+      end
+
+      def call(env)
+        start_time = Time.current
+
+        @app.call(env).on_complete do |response_env|
+          log_tags = {
+            jti: jti(env, response_env),
```

Comment:
What's jti? 

Might be worth spelling out directly if you think someone like me who doesn't know this stands for could someday be reading these logs.

Reply:
i think it's a common enough nomenclature for anyone familiar with JWT. It's basically a unique identifier for the JWT session. I could put a comment around it if helpful.

Reply:
Whoops. My bad. No need.

Reply:
I went ahead and added a bunch of YARD documentation to the logging middleware. Should help document JTI and other attributes a bit better.